### PR TITLE
Move to OperatorSDK 1.33.0 using kustomize/v2

### PR DESCRIPTION
**Pull request by tkrishtop**

### CODE REVIEW

File: internal/policy/operator/testdata/disconnected_bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: simple-disconnected-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.33.0
```
Comment (acornett21) [inline internal/policy/operator/testdata/disconnected_bundle/metadata/annotations.yaml:8]: @tkrishtop We have more test data now, could you also update those bundles in `/testdata` as well?

Comment (tkrishtop) [reply internal/policy/operator/testdata/disconnected_bundle/metadata/annotations.yaml:8]: hi @acornett21 that's fixed and the tests are green, could you please have a look one more time?

Comment (acornett21) [reply internal/policy/operator/testdata/disconnected_bundle/metadata/annotations.yaml:8]: I still only see this file committed, shouldn't there be other test files?

**Comment by tkrishtop** [comment]
hello @acornett21 is it ok to merge this PR?

**Review by acornett21 — APPROVED** [review]

**Comment by tkrishtop** [comment]
Thank you, @acornett21! I don't have the write permissions and cannot merge pull requests. Please go ahead and merge it whenever it's fine for you. Appreciate your help!

**Review by komish — APPROVED** [review]
/lgtm

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**